CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 # [Unreleased]
 
 * Update `rand` to 0.4 and `gcc` 0.3 to `cc` 1.0. (`rand` 0.5 exists but has a lot of breaking changes and no longer compiles with 1.14.0.)
+* Disallow messages that are 0 or overflow the curve order; this should never happen (probability < 2^128) with a random hash, and 0 in particular is cryptographically dangerous.
```

Comment:
Technically you can still create a bad Message via the From<[u8; 32]> impl, which really sucks, tbh. Can we convert that to a non-trait from function that returns a Result? I know its an API change which sucks but we might as well wait for the static context from upstream to make the from_slice() check free anyway?